dashboard/settings.py

```diff
@@ -25,6 +29,26 @@
     os.path.join(os.path.dirname(__file__), ".."),
 )
 
+
+def apply_env_overrides(env: Dict[str, Any], environ: os._Environ) -> Dict[str, Any]:
+    """
+    Replaces values for any keys in env found in the environment
+    """
+    env_copy = env.copy()
+    for key in env_copy.keys():
+        if key in environ:
+            os_value = environ[key]
+            try:
+                os_value = json.loads(os_value)
+                logger.debug('Value was valid JSON; saved parsed data in env object.')
+            except json.JSONDecodeError:
+                pass
```

Comment:
is the "pass" used for non-json values only?

otherwise, I think we need to log error for malformed json value.

Reply:
So the logic here is,

1) grab the value from the environment variable, store it in a variable
2) try to parse the value as JSON, if it works, reassign it to the variable
3) if it doesn't work, no sweat, it's probably just a vanilla string value.

I don't think I'd want to log an error since in many cases, it won't be valid JSON, and that's expected. I can add a debug log message if you'd like so the logic is more visible when testing. Would you like that?

Reply:
Yes. If you can just add those comment lines in

Reply:
I added a debug log message. Does that work? https://github.com/tl-its-umich-edu/my-learning-analytics/pull/1425/commits/d15452f14db9a7c6efe3a4760baff0221f451d88